coding-standards.php

```diff
@@ -0,0 +1,222 @@
+    <div class="carousel-inner" role="listbox">
+        <div class="item active">
+            <?php if (get_field("glossary_content_banner_image_url_path") and get_field("glossary_content_banner_image_url_upload")): ?>
+                <!-- 1st, If both values exist, use the uploaded file path-->
+                <img src="<?php the_field('glossary_content_banner_image_url_upload'); ?>" alt="" />    
+            <?php else: ?>      
+                <?php if (get_field("glossary_content_banner_image_url_upload")): ?>
+                    <!-- 2nd, both values don't exist, so if the upload value exists, use this-->
+                    <img src="<?php the_field('glossary_content_banner_image_url_upload'); ?>" alt="" />    
+                <?php elseif (get_field("glossary_content_banner_image_url_path")): ?>
+                    <!-- 3rd, both values don't exist, so if the text path value exists, use this-->
+                    <img src="<?php echo do_shortcode('[ghcmThemeImg]'); ?><?php the_field('glossary_content_banner_image_url_path'); ?>"  alt="" /> 
+                <?php else: ?>
+                    <!--4th, if no banner image is specified either by text or file upload(image gallery image), then use default image fallback -->
+                    <img src="<?php echo do_shortcode('[ghcmThemeImg]'); ?>services/healthcare-mgmt/banner.jpg" alt="" />        
+                <?php endif; ?>    
+            <?php endif; ?>    
+            <div class="carousel-caption">            
+            </div>
+        </div>
+    </div>
+    <span class="contentRespBannerLeftEdge"></span>
+    <span class="contentRespBannerRightEdge"></span>
+</div>
+<script type="text/javascript">
+    alignBannerImage();
+</script>
+
+<?php
+$page_header = get_field('glossary_page_header_title');
+
+if (have_rows('glossary_page_breadcrumb')) {
+    $bread_crumb_html = '<div class="row"><div class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><div class="clear full breadcrumbs">';
+    $bread_crumb_html .= '<ul class="ghn breadcrumbs">';
+    // loop through the rows of data
+    $index_of_loop = 0;
+    $number_of_rows = count(get_field('glossary_page_breadcrumb'));
+    while (have_rows('glossary_page_breadcrumb')) : the_row();
+        //determine if the row is the last row (set "last" class for removing breadcrumb carrot)
+        if ($index_of_loop == ($number_of_rows - 1)) {
+            $bread_crumb_html .= '<li class="last">' . get_sub_field('glossary_repeater_level') . '</li>';
+        } else {
+            $bread_crumb_html .= '<li>' . get_sub_field('glossary_repeater_level') . '</li>';
+        }
+        $index_of_loop++;
+    endwhile;
+    $bread_crumb_html .= '</ul>';
+    $bread_crumb_html .= '</div></div></div>';
+}
+
+$glossary_content_AE = "";
+$glossary_content_FJ = "";
+$glossary_content_KO = "";
+$glossary_content_PT = "";
+$glossary_content_UZ = "";
+$glossary_content_num = "";
+
+$args_glossary_items = array(
+    'posts_per_page' => -1,
```

Comment:
It would be nice to line these up:

```
`posts_per_page' => -1,
'post_type'           => 'glossary',
'orderby'              => 'title',
// ...
```